soot-infoflow-summaries/summariesManual/System.Security.SecureString.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" ?>
+<summary fileFormatVersion="101">
+	<methods>
+		<!-- This is not entirely correct, this is a char*, but we cannot handle this in Jimple properly yet -->
```

Comment:
Dann ist ``char`` aber die schlechteste Lösung. Ein ``char`` ist ein Byte, ein Pointer sind 8 Byte. Ich würde eher einen allgemeinen Pointer-Typ in Jimple einführen und die verschiedenen spezialisierten Pointer (``char*``, ``int*``, usw.) davon ableiten.